File: src/Python/easy/array/problem1588.py
from typing import List

class Solution:
    def sumOddLengthSubarrays(self, arr: List[int]) -> int:
        subArr = []
        for i in range(1, len(arr) + 1, 2):
           for j in range(len(arr)):
               if j + i <= len(arr):
                  subArr.append(sum(arr[j:i + j]))
        return sum(subArr)



arr = [1,4,2,5,3]

solution = Solution()
solution.sumOddLengthSubarrays(arr)

File: src/JavaScript/easy/array/problem1588.js
/**
 * @param {number[]} arr
 * @return {number}
 */

var sumOddLengthSubarrays = function (arr) {
    let subArr = []
    for(let i = 1; i <= arr.length; i += 2){
        for(let j = 0; j < arr.length; j++){
            if(j + i <= arr.length){
                subArr.push(...arr.slice(j,i + j))
            }
        }
    }
    return subArr.reduce((acc, item) => acc + item)
};

// let arr = [1,4,2,5,3]
// let arr = [1,2]
let arr = [10,11,12]

sumOddLengthSubarrays(arr)